spec/lib/spotlight/dor/indexer_spec.rb

```diff
@@ -222,38 +206,33 @@
 
     describe '#add_general_notes' do
       before do
-        allow(r).to receive(:mods).and_return(mods)
         subject.send(:add_general_notes, sdb, solr_doc)
       end
 
       context 'no general notes, but other types of notes' do
-        let(:mods) do
-          Nokogiri::XML <<-EOF
-            <mods xmlns="#{Mods::MODS_NS}">
-              <note displayLabel="preferred citation">(not a document subtype)</note>
-              <note displayLabel="Document subtype">memorandums</note>
-              <note displayLabel="Donor tags">Knowledge Systems Laboratory</note>
-            </mods>
-            EOF
+        let(:modsbody) do
+          <<-EOF
+            <note displayLabel="preferred citation">(not a document subtype)</note>
+            <note displayLabel="Document subtype">memorandums</note>
+            <note displayLabel="Donor tags">Knowledge Systems Laboratory</note>
+          EOF
         end
 
         it 'is blank' do
-          expect(solr_doc['general_notes_ssim']).to be_blank
+          expect(solr_doc).not_to include 'general_notes_ssim'
         end
       end
 
       context 'ignore extra notes' do
-        let(:mods) do
-          Nokogiri::XML <<-EOF
-            <mods xmlns="#{Mods::MODS_NS}">
-              <note displayLabel="Document subtype">memorandums</note>
-              <note>a generic note</note>
-            </mods>
-            EOF
+        let(:modsbody) do
+          <<-EOF
+            <note displayLabel="Document subtype">memorandums</note>
+            <note>a generic note</note>
+          EOF
         end
 
         it 'extracts the doc subtypes' do
-          expect(solr_doc['general_notes_ssim']).to contain_exactly 'a generic note'
```

Comment:
This change seems fine, but I think we should keep our pull requests as small as possible, which means, not changing specs unrelated to what is been changed in the code.  If you want to change the specs, you should do that in a separate pull request.